docs/FAQ.md

```diff
@@ -1,11 +1,159 @@
 # TrenchBoot FAQ
 
-1. [Why does TrenchBoot use an intermediate launcher?](#1-why-does-trenchboot-use-an-intermediate-launcher)
-2. [What are the benefits of measurement over signature validation?](#2-what-are-the-benefits-of-measurement-over-signature-validation)
-3. [What do I need to incorporate TrenchBoot into my system?](#3-what-do-i-need-to-incorporate-trenchboot-into-my-system)
-4. [Where do I start if I want to help with contributions?](#4-where-do-i-start-if-i-want-to-help-with-contributions)
+1. [Why use Trenchboot?](#1-why-use-trenchboot-background-info)
+2. [How does TrenchBoot work?](
+    #2-how-does-trenchboot-work-trenchboot-architecture)
+3. [How to use Trenchboot?](#3-how-to-use-trenchboot-practical-usage)
+4. [Why does TrenchBoot use an intermediate launcher?](
+    #4-why-does-trenchboot-use-an-intermediate-launcher)
+5. [What are the benefits of measurement over signature validation?](
+    #5-what-are-the-benefits-of-measurement-over-signature-validation)
+6. [What do I need to incorporate TrenchBoot into my system?](
+    #6-what-do-i-need-to-incorporate-trenchboot-into-my-system)
+7. [Where do I start if I want to help with contributions?](
+    #7-where-do-i-start-if-i-want-to-help-with-contributions)
+8. [Where to find the sources?](#8-trenchboot-sources)
+9. [Various GitHub and further documentation links](
+    #9-various-github-and-further-documentation-links)
 
-## 1. Why does TrenchBoot use an intermediate launcher?
+## 1. Why use Trenchboot? (background info)
+
+### What is Trust?
+
+Trust is when the computer does something on our behalf and we have to trust it
+with our data. A trusted component is that which always behaves in an expected
+manner.  To make a component "trusted", you must establish some root of trust.
+A Root of Trust Measurement is the process of measuring (hashing) components on
+a system to validate their integrity. When establishing a root of trust, there
```

Comment:
https://github.com/TrenchBoot/TrenchBoot.github.io/blob/cleanup/docs/theory/Glossary.md#trust has different, incompatible definition. Also, there is no `Root of Trust Measurement`, but there is `Root of Trust for Measurements`. It is not a process of measuring, though.

Please add a description that is consistent with the glossary.

Reply:
On one hand, an explanation more verbose than a dictionary-like definition is more friendly for someone who would actually ask the question in the section header. On the other hand, it's not good that we try to define a term in two seperate places, and they are entirely different. Such small things may leave a bad opinion on a project that is all about trust. 

And as you said, it does sound wrong to measure the Root of Trust. We can not do that, otherwise it would not be the Root. We can determine if we trust something else by forming a chain of transient trust, like in PGP or x509.

Reply:
Also, I think if the glossary defines such basic concepts like trust, trustee, trustor, it could also use defining the basic terms related to cybersecurity like Integrity, Confidentiality, Authenticity, Non-Repudiation, and basic techniques related to it, like a Hash, a Signature, a Certificate.

Reply:
Not really, this isn't PKI, it is more generalized concept. I like the beginning, but please drop everything from `To make a component "trusted"` to `is identical to the one our Root of Trust assured we can trust by signing it.`

Also, `Static Root of Trust Measurements` -> `Static Root of Trust for Measurement` (add `for`, make `Measurement` singular), same for DRTM.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/0463f5e6c506ce0ba96b957b98d8768e5a12415d..25021ec141d2a88fe039a75c8743ba4dd733f2e3